e2e-tests/Jenkinsfile

```diff
@@ -0,0 +1,148 @@
+import groovy.json.JsonOutput
```

Comment:
The agreement when we put the e2e tests in same repository with the rest of Socorro is that the e2e tests would be entirely in this directory so as not to interfere with the rest of Socorro's infrastructure.

So for now, it needs to be in this file and not in the repository root.